library/std/src/lib.rs

```diff
@@ -340,6 +340,8 @@
 #![feature(thread_local_internals)]
 //
 #![default_lib_allocator]
+//
+#![feature(io_null)]
```

Comment:
this should go under "only for re-exporting" I think

Reply:
Fixed. :)